local-server.md

```diff
@@ -0,0 +1,182 @@
+#### Setting up a local server
+
+Assumptions:
+  - You're using a recent Ubuntu for install, below examples developed on a qemu VM using 24.04.
+    - Examples below use `apt`
+    - Has Python 3.12, `git` already installed
+  - You are going to use the server on your LAN for local installs, and not expose it to the internet, hence no discussion of proxies or whatnot.
+
+First check IPv6 connectivity from your VM:
+```bash
+curl -6 https://sysupgrade.openwrt.org/json/v1/overview.json
+```
+
+If that fails to connect, then you will have all sorts of issues unless you resolve them.  The easiest thing to do is just disable IPv6 on your VM:
+```bash
+sudo vi /etc/sysctl.d/10-ipv6-privacy.conf
+```
+Add one line:
+```
+net.ipv6.conf.all.disable_ipv6 = 1
+```
+and reload:
+```bash
+sudo sysctl -f /etc/sysctl.d/10-ipv6-privacy.conf
+```
+
+If you can figure out how to get qemu to punch through the IPv6 blocking, @efahl would really (really) like to know.
+
+Make sure you have `podman`, Ubuntu 24.04 did not:
+
+```bash
+cd ~
+sudo apt -y install podman
+systemctl --user enable podman.socket
+systemctl --user start podman.socket
+systemctl --user status podman.socket
+```
+
+Test that podman is working by running the Alpine Linux container.  This will drop you on its command line, and if you see the `/ #` prompt, simply `exit` to return to your server's command line.
+
+    podman run --rm -it docker.io/library/alpine:latest
+    / # cat /etc/os-release
+    NAME="Alpine Linux"
+    ID=alpine
+    VERSION_ID=3.20.3
+    PRETTY_NAME="Alpine Linux v3.20"
+    HOME_URL="https://alpinelinux.org/"
+    BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
+    / # exit
+        
+
+Create a new Python virtual environment using `venv`:
+
+```bash
+sudo apt -y install python3-venv
+python3 -m venv asu-venv
+. asu-venv/bin/activate
+```
+
+Test your new virtual environment.  Verify that the executables are in your venv, and that the Python version is 3.11 or newer.
```

Comment:
Isn't it easier to just install `python3-poetry` and let poetry handle that? Or install `podman-compose` for that matter...?